src/app/campaign/availability/availability.component.ts

```diff
@@ -117,6 +126,23 @@ export class AvailabilityComponent {
     );
   }
 
+  getFlightErrors() {
```

Comment:
suggestion: slight refactor to get function and in the template using `ngIf` with `as`:
```
<ul class="errors" *ngIf="errors as flightErrors">
        <li class="error" *ngFor="let error of flightErrors">
```
```
get errors() {
    const errors = [];

    if (this.allocationPreviewError) {
      errors.push(`Got error ${this.allocationPreviewError.status} from allocation preview.`);
    }

    if (this.flight.status_message) {
      errors.push(this.flight.status_message);
    }

    return errors;
  }
```

Reply:
This looks much cleaner. I always seem to forget about setters/getters. I pushed up a change with this suggestion. Thank you.

Reply:
Looks great, thanks!